src/AlertsAggregator.py

```diff
@@ -0,0 +1,222 @@
+import pandas as pd
+import matplotlib.pyplot as plt
+import warnings
+warnings.filterwarnings('ignore')
+from datetime import datetime
+from models import AlertCountPerDay
+from Errors import WrongSettlementException
+from datetime import time, datetime, timedelta
+import requests
+GEONAMES_URL = "http://api.geonames.org/searchJSON"
+
+
+class AlertsAggregator:
+    def __init__(self, df):
+        self.df = df
+
+    # This func get dataframe and return a set of all the settlement in df
+    def return_data_list(self):
+        items_lst = self.df['data'].tolist()
+        items_lst_no_duplications = set(items_lst)
+        return items_lst_no_duplications
+
+
+    # This function get dataframe and settlement_name and return a list of all the settlement in the df that settlement_name is insid them
+    def create_user_settl_lst(self, settlement: str):
+        given_lst = self.return_data_list()
+        settl_lst = []
+        for item in given_lst:
+            if settlement.lower() == item or f"{settlement.lower()}, " in item:
+                return [item]
+            elif f"{settlement.lower()} " in item  or f" {settlement.lower()}" in item or settlement.lower() in item :
+                if item not in settl_lst:
+                    settl_lst.append(item)
+        if settl_lst != []:
+            raise WrongSettlementException(f"Incorrect settlement, please try again input one of this options: {settl_lst}")
+        return settl_lst
+
+    
+    # This function use geonames api and return True if the user settlement input is actually exist else returns False
+    def does_the_settlement_exist(self, settlement: str):
```

Comment:
```suggestion
    def is_real_settlement(self, settlement: str):
```

Reply:
also use Type annotations all over you code.
(in this case you need return type annotation)


```suggestion
    def does_the_settlement_exist(self, settlement: str)-> bool :
```